app/styles/main.scss

```diff
@@ -133,45 +147,30 @@ padding-bottom: 50px;
   margin-top: 18px;
 }
 
-#form-spacing {
-  background-color: #fff;
-  background-image: none;
-  border: 1px solid #ccc;
-  border-radius: 4px;
-  box-shadow: inset 0 1px 1px rgba(0,0,0,.075);
-  color: #555;
-  display: block;
-  width: 50%;
-  height: 34px;
-  padding: 6px 12px;
-  font-size: 14px;
-  line-height: 1.42857143;
-  -o-transition: border-color ease-in-out .15s,box-shadow ease-in-out .15s;
-  transition: border-color ease-in-out .15s,box-shadow ease-in-out .15s;
-  -webkit-box-shadow: inset 0 1px 1px rgba(0,0,0,.075);
-  -webkit-transition: border-color ease-in-out .15s,-webkit-box-shadow ease-in-out .15s;
+.pad-head {
+  padding: 30px 10px 0px 10px;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `30px 10px 0px` instead of `30px 10px 0px 10px`<br>`0px` should be written without units as `0`